runtime/flang/ftni64.h

```diff
@@ -16,7 +16,7 @@
 
 #define __HAVE_LONGLONG_T
 
-#if defined(LINUX8664) || defined(OSX8664) || defined(TARGET_LLVM_ARM64)
```

Comment:
This patch not only replaces but also removes some of the uses of LINUX8664. was that intentional? If so, it needs to be mentioned in the commit message...

Reply:
Yes, it's intentional. I have updated the commit message. Is it better?